distro/configuration/concepts/lab_tests_demo-package.csv

```diff
@@ -0,0 +1,116 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Absolute low,Critical low,Normal low,Normal high,Critical high,Absolute high,Units,Allow decimals,Display precision,_version:1,_order:1000
+5092AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,CIEL:5092;AMPATH:5092;SNOMED CT:431314004,Arterial blood oxygen saturation (pulse oximeter),Saturation d'oxygène dans le sang,SpO2,,"Oxygen saturation, as measured from a peripheral oximeter.",,Test,Numeric,0,89,92,,,100,%,Yes,,,
```

Comment:
Where we have CIEL-derived content, it would be better to do this as an OCL package rather than using the concepts domain... As a general rule, for the RefApp, we want to prefer OCL packages to the Iniz concept domain where possible.

Reply:
I agree with you @ibacher, however for cases where concept modifications/edits where necessary iniz concept domain would come in handy, say adding new mappings, translations etc.

Reply:
So, all of those types of things are possible in OCL. But really, my concern is that if we are using concepts that are mapped to CIEL then we can end up with duplication of those concepts in both the OCL packages included in the RefApp and these files, which is a potential mess of inconsistencies. For example, this concept is _already_ included in the reference application.

Reply:
Hi @ibacher, how are the concepts coming through the OCL package managed/versioned? Also would I have to correct this for all the remaining CIEL mapped concepts already existing in this iniz domain?

Reply:
> how are the concepts coming through the OCL package managed/versioned? 

The OCL app itself has a concept of versions for both sources and collections. They're versioned by releases coordinated using OCL itself.

> Also would I have to correct this for all the remaining CIEL mapped concepts already existing in this iniz domain?

Not a priority... just for new content being added.

Reply:
Mostly the reason for doing this is to ensure that the OCL module is able to find the existing concepts. It tracks metadata about concepts that have been loaded through the OCL module so that mappings in OCL can point to the right concept.

Reply:
Thanks @ibacher for the heads up. I've gone through the [getting started with OCL dictionary manager](https://wiki.openmrs.org/display/projects/User+Guide%3A+OpenMRS+Dictionary+Manager) and I see I needed access to the dictionary manager. Could you please grant me access so I can create the demo packages? Thanks.

Reply:
You should be able to register yourself here: https://app.openconceptlab.org/#/accounts/signup

Reply:
@ibacher I've completed registration however, cannot search concepts. Would I be missing some privileges?